Firestore/core/src/firebase/firestore/local/leveldb_migrations.cc

```diff
@@ -52,8 +55,10 @@ namespace {
  *   * Migration 3 deletes the entire query cache to deal with cache corruption
  *     related to limbo resolution. Addresses
  *     https://github.com/firebase/firebase-ios-sdk/issues/1548.
+ *   * Migration 4 ensures that every document in the remote document cache
+ *     has a sentinel row with a sequence number.
  */
-const LevelDbMigrations::SchemaVersion kSchemaVersion = 3;
+const LevelDbMigrations::SchemaVersion kSchemaVersion = 4;
```

Comment:
Is it worth attempting to keep the migration numbers in sync with the other clients?

https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/src/local/indexeddb_schema.ts#L39

Reply:
We would have to implement held-write-acks removal before this, which I don't think is a worthwhile delay.